swift/utils/logger.py

```diff
@@ -10,10 +10,9 @@
     '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
 
 
-def is_master(group=None):
-    from torch import distributed as dist
-    return dist.get_rank(
-        group) == 0 if dist.is_available() and dist.is_initialized() else True
+def is_master():
```

Comment:
Please move to torch_utils

Reply:
There is the problem of recursive import need to be solved